src/style.css

```diff
@@ -0,0 +1,102 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 30px 14px 14px;
+  display: flex;
+  flex-direction: column;
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 45px;
+  align-self: center;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin-bottom: 5px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+}
+
+.card__rate,
+.card__price {
+  margin-bottom: 20px;
+}
+
+.card__rate {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.rate__stars {
+  display: inline-flex;
+  width: 100px;
+}
+
+.stars__item {
+  height: 15px;
+  width: 15px;
+  background-image: url("images/star.svg");
+  margin-right: 4px;
+}
+
+.star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
Write rules for all the modifiers, so that I could change class to stars--1/ stars--3 in HTML, and number of active stars would change